packages.yml

```diff
@@ -20,7 +20,7 @@ fresnel:
 hoomd:
   ignore: beta,alpha,rc
   repository: https://github.com/glotzerlab/hoomd-blue
-  version: v3.8.1
+  version: fix-rocm-build
```

Comment:
Should we change this to trunk-minor now?

Reply:
Yes, and then 3.9.0 after the hoomd 3.9 release and before merging this branch.